src/models/Transaction.js

```diff
@@ -104,4 +104,5 @@ export type SmartWalletTransferTransaction = {
   hash: string,
   value: Object,
   asset: string,
+  status: string,
```

Comment:
Why do you need this? For PPN transactions we don't have status, as they're getting confirmed immediately

Reply:
After typing, flow complains on this line: https://github.com/pillarwallet/pillarwallet/blob/42e49d3fcd3800e67dcea137125633fe784d2773/src/utils/smartWallet.js#L53

![image](https://user-images.githubusercontent.com/263335/63302802-75eed900-c2b4-11e9-920a-35bc0dc3dcec.png)

Any suggestion?

Reply:
Cool, then leave the `status` field in typing